tests/unit_tests/ensemble_evaluator/test_async_queue_execution.py

```diff
@@ -65,12 +74,21 @@ async def test_happy_path(
 
     assert mock_ws_task.done()
 
-    event_0 = from_json(mock_ws_task.result()[0])
-    assert event_0["source"] == "/ert/ensemble/ee_0/real/0"
-    assert event_0["type"] == "com.equinor.ert.realization.waiting"
-    assert event_0.data == {"queue_event_type": "WAITING"}
-
-    end_event_index = len(mock_ws_task.result()) - 1
-    end_event = from_json(mock_ws_task.result()[end_event_index])
-    assert end_event["type"] == "com.equinor.ert.realization.success"
-    assert end_event.data == {"queue_event_type": "SUCCESS"}
+    if FeatureToggling.is_enabled("scheduler"):
```

Comment:
Look here for the change of events emitted by the new local driver. Also note that "waiting" is translated to "SUBMITTED", but not for the legacy driver/job_queue_node(?).

Reply:
I think this is because the initial state is assumed to be `WAITING` in Scheduler, so the first new event is `SUBMITTED`, while JobQueue starts with `NOT_ACTIVE`, so the first event is `WAITING`.

I don't see the need for two different states that both say "we're not doing anything yet". Maybe we should consider removing `NOT_ACTIVE` from JobQueue to avoid this `if`? Alternatively, Scheduler might send `WAITING` explicitly.

Reply:
I am fine with differing behaviour for now, so we can reduce the number of states while transitioning to Scheduler. Lets reconsider if a case pops up where we need to distuingish between NOT_ACTIVE and WAITING in Scheduler.